build/ci/unit-tests-template.yml

```diff
@@ -0,0 +1,48 @@
+# Template used by unit-tests.yml
+
+jobs:
+- job: ${{ parameters.jobName }} ${{ parameters.buildConfiguration }}
```

Comment:
This is no longer building debug for the en-US locale.

Reply:
I don't follow - that's what the first job does in unit-tests.yml.

Reply:
My mistake. The first item appears higher up in between two large deleted chunks.